apps/src/templates/sharing.html.ejs

```diff
@@ -55,22 +55,35 @@
       <!-- Mount point for DownloadReplayVideoButton component. -->
       <span id="download-replay-video-container"></span>
     <% } %>
-    <% if (options.sendToPhone) { %>
-      <button id="sharing-phone">
-        <i class="fa fa-mobile fa-lg"></i>
-        <%= msg.sendToPhone() %>
-      </button>
-    <% } %>
+    <button id="sharing-phone">
+      <i class="fa fa-mobile fa-lg"></i>
+      &nbsp;
+      <%= msg.sendToPhone() %>
+    </button>
     <div id="download-replay-video-error" style="font-size: 12px; display: none;">
       <%= msg.downloadReplayVideoButtonError() %>
     </div>
   </div>
-  <div id="send-to-phone" style="display: none">
-    <label for="phone">Enter a US phone number:</label>
-    <input type="tel" id="phone" name="phone" />
-    <button id="phone-submit" onClick="return false;">Send</button>
-    <div id="phone-charges">A text message will be sent via <a href="http://twilio.com" target="_blank">Twilio</a>. Charges may apply to the recipient.</div>
-  </div>
+
 <% } %>
 
 </div>
+
+<% /* Always show the "Send to phone" button, and when it's clicked to toggle
+    * the section below into view, the QR Code, but only show the phone number
+    * in that section below when options.isUS is true, which is done
+    * for US users. */ %>
+<div id="send-to-phone" style="display: none">
+  <% if (options.isUS) { %>
+    <div id="send-to-phone-left">
+      <label for="phone">Enter a US phone number:</label>
+      <input type="tel" id="phone" name="phone" />
+      <button id="phone-submit" onClick="return false;">Send</button>
+      <div id="phone-charges">A text message will be sent via <a href="http://twilio.com" target="_blank">Twilio</a>. Charges may apply to the recipient.</div>
+    </div>
+  <% } %>
+  <div id="send-to-phone-right">
+    <label>Scan this code with your phone camera:</label>
```

Comment:
can we translate this?

Reply:
Oops yes.  Good catch.  Updated in https://github.com/code-dot-org/code-dot-org/pull/33902.

Reply:
thanks!